src/quote/indexQuoteProvider.ts

```diff
@@ -94,38 +88,14 @@ export class FlashMintQuoteProvider
     const network = await provider.getNetwork()
     const chainId = network.chainId
     switch (contractType) {
-      case FlashMintContractType.wrapped: {
-        const wrappedQuoteProvider = new WrappedQuoteProvider(provider)
-        const wrappedQuote = await wrappedQuoteProvider.getQuote(request)
-        if (!wrappedQuote) return null
-        const builder = new WrappedTransactionBuilder(provider)
-        const txRequest: FlashMintWrappedBuildRequest = {
-          isMinting,
-          indexToken: indexToken.address,
-          inputOutputToken: inputOutputToken.address,
-          inputOutputTokenSymbol: inputOutputToken.symbol,
-          indexTokenAmount,
-          inputOutputTokenAmount: wrappedQuote.inputOutputTokenAmount,
-          componentSwapData: wrappedQuote.componentSwapData,
-          componentWrapData: wrappedQuote.componentWrapData,
-        }
-        const tx = await builder.build(txRequest)
-        if (!tx) return null
-        return {
-          chainId,
-          contractType,
-          contract: contractAddress,
-          isMinting,
-          inputToken,
-          outputToken,
-          indexTokenAmount,
-          inputOutputAmount: wrappedQuote.inputOutputTokenAmount,
-          slippage,
-          tx,
-        }
-      }
```

Comment:
I would think that we won't use `WrappedQuoteProvider` ever because of `ERC4626QuoteProvider`. So removed it everywhere.

Let me know if it's better to keep it in for now.